sstables/storage.cc

```diff
@@ -114,6 +117,38 @@ future<> filesystem_storage::rename_new_file(const sstable& sst, sstring from_na
     });
 }
 
+future<> filesystem_storage::sync_cached_directory(std::string_view name) const {
+    std::string key { name };
```

Comment:
nit: scylla style is typicallt `std::string key(name);` or `auto key = std::string(name);`

Reply:
Thanks, will update.